Tests/Unit/Domain/Extractor/Converter/DateConverterTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+namespace Neos\MetaData\Extractor\Tests\Unit\Converter;
+
+/*
+ * This file is part of the Neos.MetaData.Extractor package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\MetaData\Extractor\Converter\DateConverter;
+use TYPO3\Flow\Tests\UnitTestCase;
+
+class DateConverterTest extends UnitTestCase
+{
+    /**
+     * @return array
+     */
+    public function gpsTimeStampDataProvider()
+    {
+        return [
+            [
+                'timeStamp' => [
+                    11.0,
+                    16.0,
+                    53.0
+                ],
+                'dateStamp' => '2016:02:05',
+                'expected' => \DateTime::createFromFormat('YmdHis', '20160205111653')
+            ]
+        ];
+    }
+
+    /**
+     * @test
+     * @dataProvider gpsTimeStampDataProvider
+     *
+     * @param array $timeStamp
+     * @param string $dateStamp
+     * @param \DateTime $expected
+     */
+    public function convertGpsDateAndTime($timeStamp, $dateStamp, $expected)
```

Comment:
I'd switch `$timeStamp` and `$dateStamp` to match `DateConverter::convertGpsDateAndTime`.